docs/next.config.js

```diff
@@ -6,4 +6,14 @@ module.exports = {
     appDir: true,
```

Comment:
I was just reading that this is no longer needed https://nextjs.org/docs/app/api-reference/next-config-js/appDir

Reply:
Nice!